RELEASE_NOTES.md

```diff
@@ -20,57 +48,116 @@
 
+  - bump jaeger-operator-2.18.4
+  #835 (@d2iq-dispatch)
+
 ### kiali
   - kiali and jaeger are now deployable with the "none" provisioner, i.e. in on-premise environments.
   #1022 (@mesosphere-mergebot)
 
-### opsportal
-  - Update opsportal to mitigate CVE-2020-1971
-  #1107 (@mesosphere-mergebot)
+  - kiali: configure to use the same version for `kiali/kialii` that matches the operator.
+  #887 (@dkoshkin)
 
-  - Update opsportal to mitigate CVE-2020-1971
-  #1106 (@armandgrillet)
+  - bump kiali-operator-1.29.1
+  #892 (@d2iq-dispatch)
 
-  - UI - resolve kubecost performance issue
-  - UI - handle license loading state, show loading instead of invalid when license is missing a status
-  #1089 (@mesosphere-mergebot)
+### opsportal
+  - fix(kommander-ui): disable addons on foundation disabled
+  - feat(kommander-ui): add license delete mutation
+  - feat(kommander-ui): replace license table with single license detail view
+  - feat(kommander-ui): allow workspace namespace to be configurable
+  #930 (@d2iq-dispatch)
 
-  - fix: update UI dependencies to mitigate CVE-2021-23337
-  #1075 (@mesosphere-mergebot)
+  - Update opsportal to mitigate CVE-2020-1971
+  #1107 (@mesosphere-mergebot)
 
   - fix: updating GitOps Source in Kommander.
   - fest: Show certification badges in Kommander project catalog.
   - feat: Allow creating Root CA secret when attaching a firewalled cluster in Kommander.
   - feat: give list of loadbalncers when attaching a firewalled cluster in Kommander.
   #1072 (@mesosphere-mergebot)
 
+  - fix: update UI dependencies to mitigate CVE-2021-23337
+  #1075 (@mesosphere-mergebot)
+
+  - Fixes bug in OpsPortal where Identity Providers would not show up (COPS-6843)
+  #953 (@d2iq-dispatch)
+
+  - Fixes bug in OpsPortal & Kommander UI where LDAP Root CA is malformed when saved
+  - Updated UI to only ship with needed dependencies
+  #964 (@d2iq-dispatch)
+
+  - fix(kommander): Fix empty non-Konvoy cluster Platform Services tab
+  #902 (@d2iq-dispatch)
+
   - fix: updating GitOps Source in Kommander.
   - fest: Show certification badges in Kommander project catalog.
   - feat: Allow creating Root CA secret when attaching a firewalled cluster in Kommander.
   - feat: give list of loadbalncers when attaching a firewalled cluster in Kommander.
   #1068 (@mesosphere-mergebot)
 
+  - Fixes bug in OpsPortal & Kommander UI where LDAP Root CA is malformed when saved
+  - Updated UI to only ship with needed dependencies
+  #976 (@d2iq-dispatch)
+
+  - UI - resolve kubecost performance issue
+  - UI - handle license loading state, show loading instead of invalid when license is missing a status
+  #1089 (@mesosphere-mergebot)
+
   - fix: updating GitOps Source in Kommander.
   - fest: Show certification badges in Kommander project catalog.
   - feat: Allow creating Root CA secret when attaching a firewalled cluster in Kommander.
   - feat: give list of loadbalncers when attaching a firewalled cluster in Kommander.
```

Comment:
this looks duplicated on L108-112?

Reply:
Yep. The PR to master had the release/4.0 milestone set.